server/plugin/api_test.go

```diff
@@ -400,22 +401,22 @@ func TestHandleUnlinkProject(t *testing.T) {
 			description: "HandleUnlinkProject: marshaling gives error",
 			body: `{
 				"organizationName": "mockOrganization",
-				"projectName": "mockProject",
+				"projectName": "mockProjectName",
 				"projectID" :"mockProjectID"
 				}`,
 			statusCode: http.StatusOK,
 			projectList: []serializers.ProjectDetails{
 				{
-					MattermostUserID: "mockMattermostUserID",
-					ProjectName:      "mockProject",
-					OrganizationName: "mockOrganization",
+					MattermostUserID: mockMattermostUserID,
+					ProjectName:      mockProjectName,
+					OrganizationName: mockOrganization,
 					ProjectID:        "mockProjectID",
 				},
 			},
 			project: serializers.ProjectDetails{
-				MattermostUserID: "mockMattermostUserID",
-				ProjectName:      "mockProject",
-				OrganizationName: "mockOrganization",
+				MattermostUserID: mockMattermostUserID,
+				ProjectName:      mockProjectName,
+				OrganizationName: mockOrganization,
 				ProjectID:        "mockProjectID",
```

Comment:
we can create a test util for `project` and for `projectList` as well by re-using the previous util something like this https://github.com/Brightscout/mattermost-plugin-servicenow/blob/e9d9ae34f20bc52872e7ab022924b582208b4ea1/server/testutils/data.go#L116